operators/src/pro/ml/xgboost.rs

```diff
@@ -497,62 +470,100 @@ mod tests {
     }
 
     #[tokio::test]
+    #[allow(clippy::too_many_lines)]
     async fn multi_tile_test() {
-        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
-        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
-        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
-        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
-        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
-        let green = make_double_raster(
-            vec![
-                255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255,
-                255, 0, 0, 0,
-            ],
+        let red = generate_raster_test_data_band_helper(
```

Comment:
why did the data chang, and what do the comment "//" at the end of the lines mean?

Reply:
I changed the input grid data to have a slightly better feeling for the training -> prediction cycle.
With this in mind, I added comments to suppress the auto-formatting. This was intended to help understand what is contained in the test data.